pallets/remote-asset-manager/src/lib.rs

```diff
@@ -945,17 +945,17 @@ pub mod pallet {
 		/// - call: The encoded call to be executed
 		/// - fee: fee (in remote currency) used to buy the `weight` and `debt`.
 		/// - require_weight_at_most: the weight limit used for the xcm transacted call.
-		fn wrap_call_into_xcm(call: Vec<u8>, require_weight_at_most: Weight, fee: u128) -> Xcm<()> {
-			let asset = MultiAsset { id: Concrete(MultiLocation::here()), fun: Fungibility::Fungible(fee) };
+		fn wrap_call_into_xcm(call: Vec<u8>, require_weight_at_most: Weight, _fee: u128) -> Xcm<()> {
+			// let asset = MultiAsset { id: Concrete(MultiLocation::here()), fun: Fungibility::Fungible(fee) };
 			Xcm(vec![
-				WithdrawAsset(asset.clone().into()),
-				BuyExecution { fees: asset, weight_limit: Unlimited },
+				// WithdrawAsset(asset.clone().into()),
+				// BuyExecution { fees: asset, weight_limit: Unlimited },
 				Transact { origin_type: OriginKind::SovereignAccount, require_weight_at_most, call: call.into() },
-				DepositAsset {
-					assets: All.into(),
-					max_assets: u32::MAX,
-					beneficiary: MultiLocation { parents: 1, interior: X1(Parachain(T::SelfParaId::get().into())) },
-				},
+				// DepositAsset {
+				// 	assets: All.into(),
+				// 	max_assets: u32::MAX,
+				// 	beneficiary: MultiLocation { parents: 1, interior: X1(Parachain(T::SelfParaId::get().into())) },
+				// },
```

Comment:
annotated the buy execution instructions since they're not able to be converted into xcm v1, any idea can adapt this? @mattsse

Reply:
hmm,
not immediately obvious, maybe conflicting xcm imports?

Reply:
https://github.com/paritytech/polkadot/blob/5b74841ea76195fb804a02c0336d05d9a2d3091b/xcm/src/v1/mod.rs#L450

this conversion 

```
From<xcm::v2::Xcm> for xcm::v1::Xcm
```

doesn't include `BuyExecution`, this instruction has different places in v1 and v2